test/cpp_extensions/msnpu_extension.cpp

```diff
@@ -64,7 +65,7 @@ void init_msnpu_extension() {
     "add(Tensor self, Tensor other, Scalar alpha) -> Tensor", &add_override);
   register_extension_backend_op(
     Backend::MSNPU,
-    "sum(Tensor self) -> Tensor", &sum_override);
+    "sum(Tensor self, ScalarType dtype) -> Tensor", &sum_override);
```

Comment:
Technically this isn't correct since dtype should be optional. That's due to an existing bug, filed here to be fixed separately:

https://github.com/pytorch/pytorch/issues/21416